src/ClientGenerator/buildMultiTargeting/Microsoft.Orleans.OrleansCodeGenerator.Build.targets

```diff
@@ -0,0 +1,3 @@
+<Project>
```

Comment:
Microsoft.Orleans.CodeGenerator.Build.targets should be a better naming here, since we're removing the duplicate "Orleans" from everywhere, project names, package names, assembly names, etc.

Reply:
The name aligns to the NuGet package name

Reply:
But not by our new standards, right?

Reply:
if the package name changes, we can change this. Otherwise it won't be automatically added to the project when it's installed.

Reply:
Unless we modify the name when we pack it, like:
`<PackagePath>build\$(PackageId).targets</PackagePath>`. I'm not sure that's better, though - it might be confusing.